organization.go

```diff
@@ -31,14 +31,14 @@ type Organizations interface {
 	// Delete an organization by its name.
 	Delete(ctx context.Context, organization string) error
 
-	// Capacity shows the current run capacity of an organization.
-	Capacity(ctx context.Context, organization string) (*Capacity, error)
+	// ReadCapacity shows the current run capacity of an organization.
+	ReadCapacity(ctx context.Context, organization string) (*Capacity, error)
 
-	// Entitlements shows the entitlements of an organization.
-	Entitlements(ctx context.Context, organization string) (*Entitlements, error)
+	// ListEntitlements shows the entitlements of an organization.
+	ListEntitlements(ctx context.Context, organization string) (*Entitlements, error)
```

Comment:
Should we name this `ReadEntitlements` instead? Though the name of the resource is plural, is returning a single struct. Also, if we take a look at the controllers file in Atlas for this resource, it's calling the `def show` method which in go-tfe we usually translate as ReadSomeResource method.